index.js

```diff
@@ -2,352 +2,436 @@ const imageUrl = 'https://img.freepik.com/free-photo/white-box_74190-267.jpg';
-        "title": "LED Light Strip",
-        "description": "Smart LED light strip with app control and customizable colors",
-        "image": imageUrl,
-        "price": 24.99,
-    },
-    {
-        "id": 36,
-        "title": "Digital Drawing Pad",
-        "description": "Digital drawing pad with pressure-sensitive pen and Bluetooth connectivity",
-        "image": imageUrl,
-        "price": 59.99,
-    },
-    {
-        "id": 37,
-        "title": "Streaming Microphone",
-        "description": "USB condenser microphone for streaming, podcasting, and recording",
-        "image": imageUrl,
-        "price": 79.99,
-    },
-    {
-        "id": 38,
-        "title": "Capture Card",
-        "description": "HDMI capture card for streaming and recording video games",
-        "image": imageUrl,
-        "price": 149.99,
-    },
-    {
-        "id": 39,
-        "title": "Laser Printer",
-        "description": "Wireless laser printer with fast printing and high-quality output",
-        "image": imageUrl,
-        "price": 199.99,
-    },
-    {
-        "id": 40,
-        "title": "Smart Speaker",
-        "description": "Smart speaker with voice assistant and high-quality sound",
-        "image": imageUrl,
-        "price": 89.99,
-    },
-    {
-        "id": 41,
-        "title": "Action Camera",
-        "description": "4K action camera with waterproof case and image stabilization",
-        "image": imageUrl,
-        "price": 149.99,
-    },
-    {
-        "id": 42,
-        "title": "Wi-Fi Range Extender",
-        "description": "Wi-Fi range extender with dual-band support and easy setup",
-        "image": imageUrl,
-        "price": 49.99,
-    },
-    {
-        "id": 43,
-        "title": "Bluetooth Tracker",
-        "description": "Bluetooth tracker for finding keys, bags, or other items",
-        "image": imageUrl,
-        "price": 29.99,
-    },
-    {
-        "id": 44,
-        "title": "Smart Alarm Clock",
-        "description": "Smart alarm clock with sunrise simulation and voice assistant",
-        "image": imageUrl,
-        "price": 69.99,
-    },
-    {
-        "id": 45,
-        "title": "Digital Pen",
-        "description": "Digital pen with smart note-taking and drawing features",
-        "image": imageUrl,
-        "price": 79.99,
-    },
-    {
-        "id": 46,
-        "title": "Noise-Isolating Earphones",
-        "description": "In-ear noise-isolating earphones with superior sound quality",
-        "image": imageUrl,
-        "price": 59.99,
-    },
-    {
-        "id": 47,
-        "title": "Laptop Cooling Pad",
-        "description": "Laptop cooling pad with adjustable fan speed and RGB lighting",
-        "image": imageUrl,
-        "price": 29.99,
-    },
-    {
-        "id": 48,
-        "title": "Smart Bicycle Lock",
-        "description": "Smart bicycle lock with GPS tracking and app control",
-        "image": imageUrl,
-        "price": 99.99,
+// const products = [
```

Comment:
Please remove these comments. Also please remember to update the API repo. They need to have the product in sync.